### ISSUE TITLE
Polyfill not working, when page is reloaded in between

### ISSUE THREAD

**Issue by mane-vg** [flackr/scroll-timeline#96]
Hello,

I am currently experiencing the following problem: 

The animations work just fine, if the page that gets loaded is scrolled all the way to the top or to the bottom.
However, if I reload the page somewhere in between, the animations don't work.

Here is an example of one of my animations.

```
drahtlosDisplay2Lower.animate(
        {
            backgroundPositionX: ['100%', '0'],
        }, {
            timeline: new ViewTimeline({
                subject: drahtlosDisplay2Lower,
                axis: 'vertical',
            }),
            delay: {phase: 'contain', percent: CSS.percent(35)},
            endDelay: {phase: 'contain', percent: CSS.percent(60)},
            fill: 'both',
        });
```

I am just about to dig deeper into the issue, but maybe some of you have experienced similar stuff?

**Comment by argyleink** [comment]
I'm seeing similar funky behavior if the page is loaded and scrolled anywhere than the top, it seems to mess up the positioning and timelines

**Comment by bramus** [comment]
Was this in Safari by any chance? The polyfill did not play nice with its Page Cache. This has been fixed in #147.

**Comment by bramus** [comment]
Assuming this was Safari. Closing. Feel free to provide more info to reopen.
